vianetwork/via-core
# Feat/via basic verifier orchestration layer

## What ❔
- Add basic orchestration layer for verifier/coordinator node.
- Update the via CLI to include the verifer and the coordinator setup
- Moved the via cli deposit cmd to the `token` (with withdraw).

## Test
1. Build the via cli
2. Run the node as verifier `make via`
3. Run the node as verifier `make via-verifier`
4. Run the node as verifier `make via-coordinator`

You should be able to start the sequencer, the verifier and coordinator services.
![Screenshot from 2024-12-17 22-04-44](https://github.com/user-attachments/assets/c7b5681c-6432-4799-8903-670fe53d9650)

5. Deposit tokens using the new cmd 
```shell
via token deposit \
    --amount 100 \
    --receiver-l2-address 0x36615Cf349d7F6344891B1e7CA7C72883F5dc049
```

## Checklist
